scripts/ae2.zs

```diff
@@ -313,11 +318,11 @@ recipes.addShaped("wct", <wct:wct>, [
 ]);
 
 // Creative Wireless Crafting Terminal
-recipes.removeByRecipeName("wct:wct");
-recipes.addShaped("wct_creative", <wct:wct_creative>, [
-    [<wct:wct>, <metaitem:field.generator.luv>],
-    [<appliedenergistics2:part:16>, <ore:gtceScrewdrivers>]
-]);
+//recipes.removeByRecipeName("wct:wct");
+//recipes.addShaped("wct_creative", <wct:wct_creative>, [
+//    [<wct:wct>, <metaitem:field.generator.luv>],
+//    [<appliedenergistics2:part:16>, <ore:gtceScrewdrivers>]
+//]);
```

Comment:
Why was this recipe commented out? The Creative Terminal should have a default recipe, so that part should remain present at minimum.